content/bucketmap.md

```diff
@@ -0,0 +1,22 @@
+## Bucket Map
+
+The Cumulus Distribution API supports bucket map, and you can get bucket map paths for a given bucket.
```

Comment:
Should we include a sentence or two about what a "bucket map" actually is and how it's used? It might be OK to assume the reader already knows, not sure.

Reply:
```suggestion
The Cumulus Distribution API supports an S3 bucket map. The `/locate` endpoint fetches bucket map paths for a given bucket.
```
Just a suggestion.